qiskit/providers/ibmq/qcircuits/__init__.py

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+
```

Comment:
can we have the folder circuits not circuits

Reply:
Sure - note this file and class is not really exposed to the user (the real binding of the user interface happens in https://github.com/Qiskit/qiskit-ibmq-provider/pull/79/files#diff-fed302e3beba659413d59e360ed8dca3R49), but I'm good changing it!